src/message_interface/mod.rs

```diff
@@ -107,12 +107,6 @@ where
     handle.listen(events, handler).await;
 }
 
-#[cfg(feature = "events")]
-/// Remove wallet event listeners, empty vec will remove all listeners
-pub async fn clear_listeners(handle: &WalletMessageHandler, events: Vec<WalletEventType>) {
```

Comment:
Can you check if we can do the same with `listen` ?

Reply:
true for java and node!

Reply:
Cool, I'm assuming we don't even handle that for Python yet so it should be fine to remove 👍🏻